src/services/codefixes/convertToAsyncFunction.ts

```diff
@@ -199,7 +199,26 @@ namespace ts.codefix {
             }
         });
 
-        return getSynthesizedDeepCloneWithRenames(nodeToRename, /*includeTrivia*/ true, identsToRenameMap, checker);
+        return getSynthesizedDeepCloneWithReplacements(nodeToRename, /*includeTrivia*/ true, original => {

```

Comment:
This was, until now, the only consumer of `getSynthesizedDeepCloneWithRenames`, so I moved the bits that had poor reusability out to here.

Reply:
weird that it had an unused optional `callback` parameter

Reply:
It was used originally, but I stopped using it a few months back when I greatly simplified convertToAsyncFunction.